install-dev/controllers/console/process.php

```diff
@@ -350,7 +350,6 @@ private function initKernel()
         }
         global $kernel;
         $kernel = new AppKernel($env, _PS_MODE_DEV_);
-        $kernel->loadClassCache();
```

Comment:
Why delete this line?

Reply:
This was an optimization not useful when using PHP 7 or above => https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/HttpKernel/Kernel.php#L394